graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphqlServiceBuilder.java

```diff
@@ -208,18 +255,47 @@ public GraphqlService build() {
         return new DefaultGraphqlService(builder.build(), dataLoaderRegistry, useBlockingTaskExecutor);
     }
 
-    private static TypeDefinitionRegistry typeDefinitionRegistry(List<File> schemaFiles) {
+    private GraphQLSchema makeSchema() {
```

Comment:
nit: We almost always use `build` instead of `make` in our code.
```suggestion
    private GraphQLSchema buildSchema() {
```